bin/env_variables_all.txt

```diff
@@ -1,77 +1,78 @@
+APPINSIGHTS_INSTRUMENTATIONKEY=SomeRandomStringForLocalDocker
+BEFTA_RESPONSE_HEADER_CHECK_POLICY=JUST_WARN
+BEFTA_S2S_CLIENT_ID=ccd_gw
+BEFTA_S2S_CLIENT_SECRET=AAAAAAAAAAAAAAAC
+BEFTA_S2S_CLIENT_SECRET_OF_AAC_MANAGE_CASE_ASSIGNMENT=AAAAAAAAAAAAAAAA
+CCD_BEFTA_CASEWORKER_1_NO_PROFILE_PWD=Pa55word11
+CCD_BEFTA_CASEWORKER_1_PWD=Pa55word11
+CCD_BEFTA_CASEWORKER_2_PWD=Pa55word11
+CCD_BEFTA_CASEWORKER_2_SOLICITOR_1_PWD=Pa55word11
+CCD_BEFTA_CASEWORKER_2_SOLICITOR_2_PWD=Pa55word11
+CCD_BEFTA_CASEWORKER_2_SOLICITOR_3_PWD=Pa55word11
+CCD_BEFTA_CASEWORKER_3_PWD=Pa55word11
+CCD_BEFTA_CASEWORKER_CAA_PWD=Pa55word11
+CCD_BEFTA_CITIZEN_2_PWD=Pa55word11
+CCD_BEFTA_CITIZEN_3_PWD=Pa55word11
+CCD_BEFTA_MASTER_CASEWORKER_PWD=Pa55word11
+CCD_BEFTA_MASTER_SOLICITOR_1_PWD=Pa55word11
+CCD_BEFTA_MASTER_SOLICITOR_2_PWD=Pa55word11
+CCD_BEFTA_MASTER_SOLICITOR_3_PWD=Pa55word11
```

Comment:
Pre existing issue:  I notice the new `befta_master.solicitor.becky` and `befta_master.solicitor.benjamin` accounts are missing from the environment variables.   However I notice that     
[aac-manage-case-assignment befta](https://github.com/hmcts/aac-manage-case-assignment/blob/master/src/functionalTest/resources/features/F-001%20-%20Assign%20Access%20within%20Organisation/F-001-Base.td.json#L15-L21) is reusing other environment references.  Is this password re-use the correct approach to reduce the number of references we keep having to add to jenkins files etc or should this file (and others) be updated to use separate references?

Reply:
Env vars for Becky and Benjamin are the correct, respective env vars. The reuse of env vars are normally avoided. If there is an example of that, it needs to be fixed.

Reply:
I will resolve this conversation and will raise separately via slack.